lib/av_client/types.ts

```diff
@@ -369,7 +374,13 @@ export type VotingRoundConfigMap = {
 }
 
 export type VotingRoundConfig = {
+  content: VotingRoundContent
+}
+
+export interface VotingRoundContent {
   reference: string
+  status: "open" | "scheduled" | "closed"
```

Comment:
Where is the actual schedule times?
Not a blocker just expect that to be returned from the endpoint on dbb

Reply:
@av-alexistoledo